File: LLM-Translations/GPT-4/avatar/Python/Go/codeforces_397_B.go

package main

import (
	"fmt"
	"strconv"
	"strings"
	"bufio"
	"os"
)

func main() {
	scanner := bufio.NewScanner(os.Stdin)
	scanner.Scan()
	t, _ := strconv.Atoi(scanner.Text())

	queries := make([][]int, t)
	for i := 0; i < t; i++ {
		scanner.Scan()
		queryStr := strings.Split(scanner.Text(), " ")
		for j := 0; j < len(queryStr); j++ {
			queryInt, _ := strconv.Atoi(queryStr[j])
			queries[i] = append(queries[i], queryInt)
		}
	}

	res := make([]string, t)
	for i, query := range queries {
		k := query[0] / query[1]
		if k*query[2] >= query[0] {
			res[i] = "Yes"
		} else {
			res[i] = "No"
		}
	}

	fmt.Println(strings.Join(res, "\n"))
}

// End of Code

File: LLM-Translations/GPT-4/avatar/Python/C/codeforces_397_B.c
#include <stdio.h>

int main() {
    int t;
    scanf("%d", &t);
    int queries[t][3];
    for (int i = 0; i < t; i++) {
        scanf("%d %d %d", &queries[i][0], &queries[i][1], &queries[i][2]);
    }
    char* res[t];
    for (int i = 0; i < t; i++) {
        int k = queries[i][0] / queries[i][1];
        if (k * queries[i][2] >= queries[i][0]) {
            res[i] = "Yes";
        } else {
            res[i] = "No";
        }
    }
    for (int i = 0; i < t; i++) {
        printf("%s\n", res[i]);
    }
    return 0;
} // End of Code
